go.mod

```diff
@@ -8,14 +8,13 @@ require (
 	github.com/ghodss/yaml v1.0.0 // indirect
 	github.com/golang/protobuf v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.6.4
-	github.com/gunk/opt v0.0.0-20190118082020-1439f71a8644 // indirect
 	github.com/knq/ini v0.0.0-20181118015158-a301e724bd35
 	github.com/knq/snaker v0.0.0-20181215144011-2bc8a4db4687
 	github.com/rogpeppe/go-internal v1.1.0
 	github.com/stretchr/testify v1.3.0 // indirect
 	golang.org/x/net v0.0.0-20181106065722-10aee1819953
 	golang.org/x/sync v0.0.0-20181221193216-37e7f081c4d4 // indirect
-	golang.org/x/tools v0.0.0-20190118193359-16909d206f00
+	golang.org/x/tools v0.0.0-20190202235157-7414d4c1f71c
```

Comment:
just curious; any reason why this is being bumped here?

Reply:
I didn't intentionally bump this, I ran `go mod tidy` because I had a large number of extra dependencies that I didn't pull in, and this was what was left. Happy to change this back though?